tdrs-backend/docker-compose.yml

```diff
@@ -52,9 +52,7 @@ services:
     volumes:
       - .:/tdpapp
     image: tdp
-    build:
-      context: .
-      dockerfile: docker/Dockerfile.local
+    build: .
```

Comment:
There's only one Dockerfile now and its in the same directory as this file so all we need to do is supply the `.` and Docker Compose will find it.